src/samples/worker/Elsa.Samples.MqttWorker/Elsa.Samples.MqttWorker.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
```

Comment:
Can you please target .NET 5 instead? We need to update the build pipeline first. Thanks!